packages/core/admin/admin/src/content-manager/components/BlocksInput/BlocksEditor.tsx

```diff
@@ -79,12 +80,14 @@ const [BlocksEditorProvider, usePartialBlocksEditorContext] =
 
 function useBlocksEditorContext(
   consumerName: string
-): BlocksEditorContextValue & { editor: Editor } {
+): BlocksEditorContextValue & { editor: Editor; modifiers: ModifiersStore } {
```

Comment:
I am not sure why do we need to pass modifiers here instead of as a value in provider? Could you please explain.

Reply:
modifiers need access to the editor to work, as it works by interacting with `Editor.marks`. I can kind of make it work by passing the editor as a param when we calling checkIsActive and handleToggle, but I have the issue of the stale editor instance that I mentioned in the other thread.

The reason to keep useBlocksEditorContext is that it has access to useSlate so the editor instance is never stale. But I agree it's a tradeoff, I wish I could do everything directly in the context